From a717773a12d78630c8501acaa2eb40b201ec0623 Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Mon, 25 Jun 2012 07:43:48 +0200
Subject: [PATCH] watchdog: Xilinx SCU driver must be used only on one CPU

Unfortunately every CPU has own SCU watchdog that's why
not possible to use it on SMP system.

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/watchdog/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/watchdog/Kconfig b/drivers/watchdog/Kconfig
index df23105..c6dd31c 100644
--- a/drivers/watchdog/Kconfig
+++ b/drivers/watchdog/Kconfig
@@ -322,7 +322,7 @@ config XILINX_PS_WATCHDOG
 
 config XILINX_SCU_WATCHDOG
 	tristate "Xilinx SCU Watchdog Timer"
-	depends on MPCORE_WATCHDOG=n
+	depends on !MPCORE_WATCHDOG && !SMP
 	help
 	  Say Y here if you want to include support for the cortex a9 scu
 	  watchdog timer in the Xilinx PS.
-- 
1.7.0.4

